frontend/src/utils/helpers/menuHelpers/setMenuToggle.tsx

```diff
@@ -0,0 +1,16 @@
+import Cookies from 'js-cookie';
+import { MENU_VALUES } from 'constants/menuPaths';
+
+export const setMenuToggle = (): boolean | undefined => {
+  const menuStatus: string | undefined = Cookies.get(MENU_VALUES.menuStatus);
+  if (menuStatus === undefined) {
+    Cookies.set(MENU_VALUES.menuStatus, 'false');
+    return false;
+  }
+  if (menuStatus === 'true') {
+    Cookies.set(MENU_VALUES.menuStatus, 'false');
+    return false;
+  }
+  Cookies.set(MENU_VALUES.menuStatus, 'true');
+  return true;
```

Comment:
Check please amount of conditions and also try to write only one `return` statement per function!
```
let isMenuOpen = true;
if ((menuStatus && menuStatus === 'true') || !menuStatus) {
  Cookies.set(MENU_VALUES.menuStatus, 'false');
}
return isMenuOpen
```

Reply:
Fixed this function and rewrote some logical. This Func should have 2 returns, because `setMenuToggle` must handle 2 conditions: